src/tainted/range.h

```diff
@@ -13,13 +13,14 @@ namespace iast {
 namespace tainted {
 class Range {
  public:
-    explicit Range(int start, int end, InputInfo *inputInfo);
+    explicit Range(int start, int end, InputInfo *inputInfo, int secureMarks);
```

Comment:
```suggestion
    explicit Range(int start, int end, InputInfo *inputInfo, secure_marks_t secureMarks);
```

Reply:
I'll also add before line 12 the type definition, e.g. `using secure_marks_t = uint32_t`. I think just for now we can use an `uint32_t` (4 bytes), later if we need to use more than 32 options then we can use a `std::bitset` which its minimum size is 8 bytes.